lib/src/main/java/com/futurewei/alcor/common/exception/NetworkKeyAllocNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.futurewei.alcor.common.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code= HttpStatus.NOT_FOUND, reason="Key allocation is not found")
```

Comment:
Thanks for adding all these exceptions!

Do you expect the same exceptions to be used by other services? If those are VPC manager specific, maybe we should move them to VpcManager?

Reply:
Thanks Liguang ~ I am not sure if there are any other microservice creating vlan , vxlan or gre. Let's talk about it